tests/services/test_feed_validator.py

```
@@ -0,0 +1,172 @@
+import asyncio
+import aiohttp
+import pytest
+from unittest.mock import patch, AsyncMock
+from src.services.feed_validator import FeedValidator
+
+@pytest.mark.asyncio
+async def test_valid_url():
```

Ruff (0.8.0)

Missing return type annotation for public function `test_valid_url`

Add return type annotation: `None`

(ANN201)